frontend/theta-project-tracker/src/app/services/export.service.ts

```diff
@@ -1,51 +1,62 @@
 import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
 import { environment } from 'src/environments/environment';
-import { Observable } from 'rxjs';
 
 @Injectable({ providedIn: 'root' })
 export class ExportsService {
 
   constructor(private http: HttpClient) { }
   apiUrl = environment.baseUrl;
   exportReportsByProjectHours(startDate, endDate, projects) {
-    console.log(projects);
+    const arrayString = JSON.stringify(this.createArray(projects));
     return this.http.get(this.apiUrl + 'export/project/hours', { params: {
       from: startDate,
       to: endDate,
-      projects: projects
-    }} );
+      projects: arrayString
+    }} ).subscribe();
   }
 
-  exportReportsByProjectCost(startDate, endDate, projects): Observable<File> {
+  exportReportsByProjectCost(startDate, endDate, projects) {
+    const arrayString = JSON.stringify(this.createArray(projects));
     return this.http.get<File>(this.apiUrl + 'export/project/cost', { params: {
       from: startDate,
       to: endDate,
-      projects: projects
-    }});
+      projects: arrayString
+    }} ).subscribe();
   }
 
-  exportReportsByUserHours(startDate, endDate, users): Observable<File> {
+  exportReportsByUserHours(startDate, endDate, users) {
+    const arrayString = JSON.stringify(this.createArray(users));
     return this.http.get<File>(this.apiUrl + 'export/user/hours', { params: {
       from: startDate,
       to: endDate,
-      users: users
-    }});
+      users: arrayString
+    }}).subscribe();
   }
 
-  exportReportsByUserCost(startDate, endDate, users): Observable<File> {
+  exportReportsByUserCost(startDate, endDate, users) {
+    const arrayString = JSON.stringify(this.createArray(users));
     return this.http.get<File>(this.apiUrl + 'export/user/cost', { params: {
       from: startDate,
       to: endDate,
-      users: users
-    }});
+      users: arrayString
+    }}).subscribe();
```

Comment:
mi itt ez az üres subscribe?
![236e17a73d0ae54a7fa423ee156c3dd5](https://user-images.githubusercontent.com/28385983/87027392-6387d180-c1dd-11ea-9cc9-55f105e7b24e.png)

Reply:
Bocsi, kiszedem.

Reply:
Egy másik branchen ott kezelem le, hogy letöltse a fájlt, amit a BE küld. De akkor kiszedtem addig innen.